app/posts/find-teacher-training/2017-12-01-alpha.md

```diff
@@ -13,12 +11,16 @@ related:
       Password: beta
   - text: "Prototype code on Github"
     href: https://github.com/DFE-Digital/search-and-compare-prototype/tree/alpha
+eleventyComputed:
+  eleventyNavigation:
+    key: find-alpha
+    title: "{{ title }}"
 ---
 
-[Alpha version 4](alpha-version-4)
```

Comment:
Why?

Reply:
Repetition in the URL slug was giving my the jitters, but have reversed these changes.